docs/hw/jetson_hookup.md

```diff
@@ -1,32 +1,89 @@
-![Hookup diagram for Jetson Nano™](data/hookup_nano2gb.jpg "Jetson Nano™ 2GB")
+## NVIDIA® Jetson Orin Nano™ Developer Kit
+
+!!! note 
+
+    ### Hookup kit
+    
+    Below lists the content of an example kit that enables mounting Jetson Orin Nano Developer Kit with some sensors like Intel® RealSense™[^3].
+
+    - Jetson Orin Nano Developer Kit
+    - Custom power cable (JST-XH to DC 5.5 mm x 2.5 mm plug)
+        - [DC power plug (5.5 mm x 2.5 mm)](https://a.co/d/cfNWywP)
+    - [USB data cable (**USB-C® to USB-C®**)](https://a.co/d/7H62wsI)
+    - Jetson NX-style developer kit mount ([STL file](./data/models/Compute/NVIDIA_Jetson/Adapter_Mount/C3-JetsonXavierNX-Mount.stl))
+    - Sensor mount plate (STL file)
+    - [M3 standoff/spacer x4](https://a.co/d/f1QWGB3)
+    - M3 screws
+    
+The [Jetson Orin Nano™ Developer Kit](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-orin/) has a 5.5 mm x 2.5 mm barrel connector jack for power (9 V to 20 V) and a **USB-C® port** for upstream data.<br>
+This can be powered from the unregulated battery port of the Create® 3 adapter board by using a JST-XH female connector to DC barrel plug cable.
 
 ## NVIDIA® Jetson Xavier NX™ Developer Kit
-The [Jetson Xavier NX™ Developer Kit](https://developer.nvidia.com/embedded/jetson-xavier-nx-devkit) has a 5.5mm x 2.5mm barrel connector jack (J16) for power (9 V to 20 V) and a USB Micro-B port (J5) for downstream data.<br>
-This can be powered from the unregulated battery port of the Create® 3  adapter board by using a JST-XH female connector to DC barrel plug cable.
-The data connection is established by using a USB Micro B to USB-C® cable.
+
+![](./data/hookup_jetson_xavier_nx_on_c3_with_PC.jpg)
+
+!!! note 
+
+    ### Hookup kit
+    
+    Below lists the content of an example kit that enables mounting Jetson Xavier NX Developer Kit with some sensors like Intel® RealSense™[^3].
+
+    - Jetson Xavier NX Developer Kit
+    - Custom power cable (JST-XH to DC 5.5 mm x 2.5 mm plug)
+        - [DC power plug (5.5 mm x 2.5 mm)](https://a.co/d/cfNWywP)
+    - [USB data cable (USB Micro-B to USB-C®)](https://a.co/d/dnlCj1d)
+    - Jetson Xavier Developer Kit mount ([STL file](./data/models/Compute/NVIDIA_Jetson/Adapter_Mount/C3-JetsonXavierNX-Mount.stl))
+    - Sensor mount plate (STL file)
+    - [M3 standoff/spacer x4](https://a.co/d/f1QWGB3)
+    - M3 screws
+    
+The [Jetson Xavier NX™ Developer Kit](https://developer.nvidia.com/embedded/jetson-xavier-nx-devkit) has a 5.5 mm x 2.5 mm barrel connector jack (J16) for power (9 V to 20 V) and a USB Micro-B port (J5) for upstream data.<br>
+This can be powered from the unregulated battery port of the Create® 3 adapter board by using a JST-XH female connector to DC barrel plug cable.
+
+The data connection is established by using a USB Micro B to USB-C® cable, which requires some software tweaking to allow the Create3-to-Jetson communication. See [this page](../setup/jetson.md).
```

Comment:
please use "communication between the Create® 3 robot and Jetson" instead of "Create3-to-Jetson communication"

Reply:
[c29e972](https://github.com/iRobotEducation/create3_docs/pull/385/commits/c29e972c7fe7440796993cd60f4e640efbff07f1)